src/test/starknet.t.sol

```diff
@@ -127,8 +127,7 @@ contract StarknetTests is DssSpellTestBase, ConfigStarknet {
 
         assertEq(daiBridge.wards(addr.addr("MCD_PAUSE_PROXY")),     1, "StarknetTest/pause-proxy-not-ward-on-dai-bridge");
         assertEq(daiBridge.wards(addr.addr("MCD_ESM")),             1, "StarknetTest/esm-not-ward-on-dai-bridge");
-        // TODO verify that this is intentional or necessary
-        assertEq(daiBridge.wards(addr.addr("STARKNET_ESCROW_MOM")), 1, "StarknetTest/escrow-mom-not-ward-on-dai-bridge");
+        assertEq(daiBridge.wards(addr.addr("STARKNET_ESCROW_MOM")), 0, "StarknetTest/escrow-mom-not-ward-on-dai-bridge");
```

Comment:
maybe change error message to "StarknetTest/escrow-mom-ward-on-dai-bridge" (remove the "not")